### DISCUSSION TITLE
Introduction of code formatter

### DISCUSSION THREAD

**Discussion by TanmayPatil105** [TanmayPatil105/procfetch#34] [Ideas]
How about adding a code formatter?
Should line wrapping be introduced?

**Comment by youpong** [comment]
I made a draft for code format.
Please take a look at my branch [youpong/procfetch@clang-format](https://github.com/youpong/procfetch/tree/clang-format)

configuration file
- src/.clang-format

formatted source files
- src/fetch.cpp
- src/color.h

Any comment is welcome.

**Comment by youpong** [comment]
In my environment it would be nice to be able to fold at 80 columns.
